Models.py

```diff
@@ -235,9 +273,16 @@ def update_cached_offline(self):
 
     def getInfoToday(self, period: int=14) -> List[float]:
         """
-        Similar to getHistoricalData but it only gets today and the last relevant days to the stock
+        This method will get the information for the stock today and the last relevant days to the stock.
+        
+        The cached_data is used so less data has to be retrieved from yf.finance as it is already being
+        held and is quicker to obtain.
```

Comment:
Can you change

as it is already being
         held and is quicker to obtain.

held to cached or something else